xpra/codecs/nvenc/encoder.pyx

```diff
@@ -2904,6 +3217,15 @@ cdef class Encoder:
             last_context_failure = monotonic()
             msg = "could not open encode session: %s" % (nvencStatusInfo(r) or r)
             log(msg)
+            
+            if r == 10:
```

Comment:
Why 10?
What does this mean?

Reply:
I can go back and dig this up, but I believe this is the scenario where you get `NV_ENC_ERR_OUT_OF_MEMORY` after you've used up all 2 encoders available on consumer cards - it looks like that is already defined, so I should just be able to replace `10` with `NV_ENC_ERR_OUT_OF_MEMORY`